book.rb

```diff
@@ -0,0 +1,15 @@
+# class for book
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] rentals should be a getter and setter variable, so kindly make it accessible with attr_accessor